BackofficeBundle/Manager/NodeManager.php

```diff
@@ -164,19 +164,18 @@ protected function getEditableStatus(NodeInterface $node = null)
      */
     public function deleteTree($nodes)
     {
-        $parentId = null;
         $siteId = $this->contextManager->getCurrentSiteId();
-        $nodeIdDeleted = array();
         foreach ($nodes as $node) {
-            $nodeId = $node->getNodeId();
-            $node->setDeleted(true);
-            $parentId = $nodeId;
-            if (!in_array($nodeId, $nodeIdDeleted)) {
-                $nodeIdDeleted[] = $nodeId;
+            $nodePath = $node->getPath();
```

Comment:
you can put that line after the test as if the node is already deleted, getting the path is useless